body/stretch_body/robot_params_RE1V0.py

```diff
@@ -0,0 +1,770 @@
+        'stall_backoff': 2.0,
+        'stall_max_effort': 20.0,
+        'stall_max_time': 0.5,
+        'stall_min_vel': 0.1,
+        'temperature_limit': 72,
+        'usb_name': '/dev/hello-dynamixel-wrist',
+        'use_multiturn': 1,
+        'retry_on_comm_failure': 1,
+        'enable_runstop': 1,
+        'disable_torque_on_stop': 1},
+    'tool_none': {
+        'use_group_sync_read': 1,
+        'retry_on_comm_failure': 1,
+        'dxl_latency_timer': 64,
+        'py_class_name': 'ToolNone',
+        'py_module_name': 'stretch_body.end_of_arm_tools',
+        'stow': {'wrist_yaw': 3.4},
+        'devices': {
+            'wrist_yaw': {
+                'py_class_name': 'WristYaw',
+                'py_module_name': 'stretch_body.wrist_yaw'}}},
+    'tool_stretch_gripper': {
+        'use_group_sync_read': 1,
+        'retry_on_comm_failure': 1,
+        'dxl_latency_timer': 64,
+        'py_class_name': 'ToolStretchGripper',
+        'py_module_name': 'stretch_body.end_of_arm_tools',
+        'stow': {'stretch_gripper': 0, 'wrist_yaw': 3.4},
+        'devices': {
+            'stretch_gripper': {
+                'py_class_name': 'StretchGripper',
+                'py_module_name': 'stretch_body.stretch_gripper'
+            },
+            'wrist_yaw': {
+                'py_class_name': 'WristYaw',
+                'py_module_name': 'stretch_body.wrist_yaw'
+            }
+        },
+        'collision_models': ['collision_stretch_gripper']},
+    'wacc':{
+        'config': {
+        'accel_LPF': 10.0,
+        'accel_range_g': 4,
+        'accel_single_tap_dur': 70,
+        'accel_single_tap_thresh': 50,
+        'ana_LPF': 10.0}},
+    'wrist_yaw':{
+        'flip_encoder_polarity': 1,
+        'gr': 2.4,
+        'id': 13,
+        'max_voltage_limit': 15,
+        'min_voltage_limit': 11,
+        'motion':{
+        'default':{
+          'accel': 8.0,
+          'vel': 3.0},
+        'fast':{
+          'accel': 10.0,
+          'vel': 5.0},
+        'max':{
+          'accel': 12,
+          'vel': 8},
+        'slow':{
+          'accel': 4.0,
+          'vel': 1.0},
+          'trajectory_max': {
+              'vel_r': 8.0,
+              'accel_r': 16.0}},
+        'pid': [640,0,0],
+        'pwm_homing': [-300,300],
+        'pwm_limit': 885,
+        'req_calibration': 1,
+        'return_delay_time': 0,
+        'stall_backoff': 0.017,
+        'stall_max_effort': 20.0,
+        'stall_max_time': 1.0,
+        'stall_min_vel': 0.1,
+        'temperature_limit': 72,
+        'usb_name': '/dev/hello-dynamixel-wrist',
+        'use_multiturn': 1,
+        'retry_on_comm_failure': 1,
+        'enable_runstop': 1,
+        'disable_torque_on_stop': 1}
+}
+# ###################### OLDER: FACTORY PARAMS #####################################################
+#The deprecated factory params dictionary for RE1.0 (5.1.2022)
+# Kept here for migration / records
+
+factory_params_deprecated = {
```

Comment:
What does this dict do? I think the "trajectory_max" param is still used by the follow_trajectory interface, and shouldn't be depreciated.

Reply:
It is just a copy of the factory_params dictionary found in 0.2.x robot_params.py:

https://github.com/hello-robot/stretch_body/blob/master/body/stretch_body/robot_params.py

It allows the migration tool to reconstruct the 0.2.x dictionary in order to do a 'diff' during migration. Params like trajectory_max aren't deprecated (they now live in the nominal dict). It is just this entire dictionary is no longer used by 0.3.x outside of the migration tool